tasks/copy_to_glacier/copy_to_glacier.py

```diff
@@ -90,7 +90,8 @@ def task(event: Dict[str, Union[List[str], Dict]], context: object) -> Dict[str,
     config = event['config']
 
     collection = config.get(CONFIG_COLLECTION_KEY)
-    exclude_file_types = collection.get(COLLECTION_META_KEY, {}).get(EXCLUDE_FILE_TYPES_KEY, [])
+    exclude_file_types = [] if collection is None else \
```

Comment:
Instead of an if statement checking for None here, what if we just changed the line above it to be:
`collection = config.get(CONFIG_COLLECTION_KEY, {})` Thoughts?

In the future, we may want to throw an error if collection key doesn't exist maybe a separate card for that as a potential bug fix? Thoughts?

Reply:
Sounds good. I'll put it on the backlog.